Tasks/UseDotNetV2/task.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "B0CE7256-7898-45D3-9CB5-176B752BFEA6",
+    "name": "UseDotNet",
+    "friendlyName": "Use .NET Core",
+    "description": "Acquires a specific version of the .NET Core SDK from the internet or the local cache and adds it to the PATH. Additionally provides proxy support.",
+    "category": "Tool",
+    "preview": true,
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "satisfies": [
+        "DotNetCore"
+    ],
+    "demands": [],
+    "ecosystem": ".NET Core",
```

Comment:
wouldn't this be `dotnet`?

Reply:
we should align with dotnet branding (iirc they are driving toward \"dotnet\"). this is probably a question for matt

Reply:
@vtbassmatt ?

Reply:
Yeah, it looks like dotnet is the go to branding here. I've updated accordingly, Matt does that seem right?

Reply:
Yes, `dotnet`